app/models/label/LabelTable.scala

```diff
@@ -81,6 +83,15 @@ object LabelTable {
   val neighborhoods = regions.filter(_.deleted === false).filter(_.regionTypeId === 2)
 
 
+  val anonId = "97760883-8ef0-4309-9a5e-0c086ef27573"
```

Comment:
No no. What I meant was to write the query that retrieves the id from the database instead of hard coding it. I mentioned the role table only because currently all the researcher user ids are also hardcoded for saving dev time. So when you update those, also update this line with the new query. Sorry for being unclear earlier.